### PR TITLE
added ci from master

### PR DESCRIPTION
**Pull request by TriplEight**

### CODE REVIEW

**Comment by TriplEight** [comment]
right after the merge I'm going to tag `1.0.0rc2`

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -153,27 +145,25 @@ build-rust-doc-release:            &build
   <<:                              *build-only
   tags:
     - linux-docker
+  before_script:
+    - sccache -s
+    - ./scripts/build.sh
```
Comment (gabreal) [inline .gitlab-ci.yml]: is it really needed to build the wasm binaries here?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -86,33 +73,38 @@ test-linux-stable:                 &test
...
+    # WASM support is in progress. As more and more crates support WASM, we
+    # should add entries here.
+    - time cargo web build -p substrate-network-libp2p
+    - time cargo web build -p substrate-panic-handler
+    - time cargo web build -p substrate-peerset
+  only:
+    - master
+    - /^[0-9]+$/
+  except:
+    - v1.0
```
Comment (gabreal) [inline .gitlab-ci.yml]: I guess the only statements exclude the excepts already.

Comment (TriplEight) [reply .gitlab-ci.yml]: it doesn't work at all since gitlab doesn't know our branches. I DMed you.

File: .gitlab-ci.yml (hunk 1 above)
Comment (TriplEight) [reply .gitlab-ci.yml]: yes

**Pull request merged**